servers/biometric_signal_sensor_interface/src/gui/templates/raw_data.html

```diff
@@ -2,25 +2,271 @@
 
 {% block mainHead %} 
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
   <!-- Plotly.js -->
   <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
 {% endblock %}
 
 {% block mainBody %}
-  {% for id in ids %}
   <div class="row">
-     <div class="col s12 m12">
-        <div id="{{ id }}"></div>
-     </div>
+    <div class="col s12 m12">
+      <div id="graph-1"></div>
+    </div>
   </div>    
-  {% endfor %}         
+  <hr>
+  <div class="row">
+    <div class="col s12 m12">
+      <div id="graph-2"></div>
+    </div>
+  </div>
+  <hr>      
+  <div class="row">
+    <div class="col s12 m12">
+      <div id="graph-3"></div>
+    </div>
+  </div> 
 
   <script>
-    var graphs = {{graphJSON | safe}};
-    var ids = {{ids | safe}};
-    for(var i in graphs) {
-        Plotly.plot(ids[i], graphs[i].data,
-          graphs[i].layout || {});
+    function rand() {
+     return Math.random();
     }
+
+   
+    var cnt = 0;
+
+    var ecg_t = [];
+    var ecg_v = [];
+    var hr_t = [];
+    var hr_v = [];
+    var hrq_t = [];
+    var hrq_v = [];
+    var rr_t = [];
+    var rr_v = [];
+    var af_time;
+    var vt_time;
+    var apc_time;
+
+    $.get("/raw_data/fetch", function(data) {
+        var res = $.parseJSON(data)
+        af_time = res['2']
+        vt_time = res['3']
+        apc_time = res['4']
+
+        if(vt_time == 0){
+          vt_time = '';
+        }
+        if(af_time == 0){
+          af_time = '';
+        }
+        if(apc_time == 0){
+          apc_time = '';
+        }
+
+        var J = $.parseJSON(res['1'])
+        var ecg = J['4113']
+        var hr = J['19']
+        var hrq = J['1000']
+        var rr = J['18']
+
+        for (var i=0;i<ecg.length;i++){
+          ecg_t.push(ecg[i][0]);
+          ecg_v.push(ecg[i][1]);
+        }
+        for (var i=0;i<hrq.length;i++){
+          hrq_t.push(hrq[i][0]);
+          hrq_v.push(hrq[i][1]);
+        }
+        for (var i=0;i<hr.length;i++){
+          hr_t.push(hr[i][0]);
+          hr_v.push(hr[i][1]);
+        }
+        for (var i=0;i<rr.length;i++){
+          rr_t.push(rr[i][0]);
+          rr_v.push(rr[i][1]);
+        }
+        console.log(rr_t.length);
+      });
+
+    Plotly.plot('graph-1', [{
```

Comment:
For the anomalies:
* change big dot for not so big inverted triangle
* position the triangle a little above the corresponding raw data value

For raw data:
Configure the y axes to show a fixed range of values (same range as the sensor providing the data)